### PR TITLE
fix(webkit): properly detect arm64 on apple silicon

### PR DESCRIPTION
**Pull request by JoelEinbinder**
`os.arch()` returns `"x64"` on all platforms. `uname -u` returns `i386` when called from node. However `sysctl` tells us the real architecture, even if we are running under Rosetta.

Depending on how we merge this into a release, we should also bump the browser number.

### CODE REVIEW

**Comment by yury-s** [comment]
Is this specific to Node 14? In Node 15 `os.arch()` returns 'arm64'.

File: src/utils/browserPaths.ts [hunk 1]
```diff
@@ -35,7 +35,10 @@ export const hostPlatform = ((): BrowserPlatform => {
     const macVersion = execSync('sw_vers -productVersion', {
       stdio: ['ignore', 'pipe', 'ignore']
     }).toString('utf8').trim().split('.').slice(0, 2).join('.');
-    const archSuffix = os.arch() === 'arm64' ? '-arm64' : '';
+    const arm64 = execSync('sysctl -in hw.optional.arm64', {
```
Comment (yury-s) [inline src/utils/browserPaths.ts:38]: Let's restrain this method to macOS 11+ and Node <15 ?

**Review by yury-s — APPROVED** [review]

File: src/utils/browserPaths.ts (hunk 1 above)
Comment (JoelEinbinder) [reply src/utils/browserPaths.ts:38]: Node 15 reports 'x64' unless you compile it yourself.

**Comment by JoelEinbinder** [comment]
as per discussion with dgozman offline, added a check that doesn't run this code unless you are on macos 11 or higher.

**Review by dgozman — APPROVED** [review]

**Pull request merged**